python/src/pycryptosat.cpp.in

```diff
@@ -997,6 +997,38 @@ static PyObject* msolve_selected(Solver *self, PyObject *args, PyObject *kwds)
     return solutions;
 }
 
+PyDoc_STRVAR(get_conflict_doc,
+"get_conflict()\n\
+Returns the conflicts in the assumptions when the last call to solve(...)\n\
+was unsatisfiable.\n\
+\n\
+:return: Which assumptions (as passed to solve(...)) are incorrect\n\
+:rtype: <dict <int,boolean>>"
+);
+
+static PyObject* get_conflict(Solver *self)
+{
+    PyObject *result = PyDict_New();
+    const std::vector<Lit> conflicts = self->cmsat->get_conflict();
+
+    for (unsigned long i = 0; i < conflicts.size(); i++) {
+        Lit lit = conflicts[i];
+        #ifdef IS_PY3K
+        PyObject *key = PyLong_FromLong(lit.var()+1);
+        #else
+        PyObject *key = PyInt_FromLong(lit.var()+1);
+        #endif
+
+        if (lit.sign() == false) {
+            PyDict_SetItem(result, key, Py_False);
```

Comment:
Do I have these swapped? I'm wondering if the semantics are:

`add_assume(-2) --> get_conflict() returns 2 == False` is a bad assumption
-- or --
`add_assume(-2) --> get_conflict() returns 2 == True` is the corrected assumption.

Does that make sense?

Reply:
add_assume? What is that? I hope that's not part of the specification because I never added that for sure. it should be that we solve with a set of assumptions. There is no add_assume.... Or shouldn't be.

Reply:
Er sorry, edited.

Reply:
Moot point when you end up returning a list rather than a dict. Resolving...